### PR TITLE
{math}[GCCcore/6.4.0] Eigen v3.3.4

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca004.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/039cf21fdd32523b5971bf48dd325417 for a full test report.

File: easybuild/easyconfigs/e/Eigen/Eigen-3.3.4-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# both in Eigen 2.x an 3.x
+include_files = ['Cholesky', 'Core', 'Dense', 'Eigen', 'Geometry', 'LU',
+                 'QR', 'QtAlignedMalloc', 'SVD', 'Sparse', 'StdVector']
+
+# only 3.x
+include_files.extend(['CholmodSupport', 'Eigenvalues', 'Householder',
+                      'IterativeLinearSolvers', 'Jacobi', 'OrderingMethods',
+                      'PaStiXSupport', 'PardisoSupport', 'SparseCholesky',
+                      'SparseCore', 'StdDeque', 'StdList',
+                      'SuperLUSupport', 'UmfPackSupport'])
```
Comment (verdurin) [inline easybuild/easyconfigs/e/Eigen/Eigen-3.3.4-GCCcore-6.4.0.eb]: Forgive my ignorance, but don't these repeat what's in the `Eigen` `easyblock`?

**Comment by JackPerdue** [comment]
I honestly don't remember why I did that   Sanity checks removed.

**Comment by verdurin** [comment]
Merging, thanks @JackPerdue

**Pull request merged**